### ISSUE TITLE
macOS m1芯片打开直接崩溃，无法使用

### ISSUE THREAD

**Issue by snodr** [wangqr/Aegisub#97]
如题。
[崩溃报告.txt](https://github.com/wangqr/Aegisub/files/6175425/default.txt)

**Comment by PutoML** [comment]
Yeah, if you click reopen it’ll eventually work. No idea why that happens.

It happens with native M1 builds, too.
